src/main/java/racingcar/RandomNumberGenerator.java

```diff
@@ -4,14 +4,18 @@
 
 public class RandomNumberGenerator {
 
+    private final String MIN_NUMBER_FORMAT = "최소값은 %s보다 작을 수 없습니다.";
+    private final String MIN_MAX_NUMBER_FORMAT = "최대값은 %s작거나 %s보다 클수 없습니다.";
+    private final String MIN_MAX_SUM_FORMAT = "최대값과 최소값에 합은 %s보다 작아야합니다.";
```

Comment:
상수 컨벤션에 따라 `static` 키워드까지 추가해보면 좋을 것 같아요.
```suggestion
    private static final String MIN_NUMBER_FORMAT = "최소값은 %s보다 작을 수 없습니다.";
    private static final String MIN_MAX_NUMBER_FORMAT = "최대값은 %s작거나 %s보다 클수 없습니다.";
    private static final String MIN_MAX_SUM_FORMAT = "최대값과 최소값에 합은 %s보다 작아야합니다.";
```